src/core/evaluator.js

```diff
@@ -1198,6 +1221,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
       var xref = this.xref;
       let parsingText = false;
       const localImageCache = new LocalImageCache();
+      const localColorSpaceCache = new LocalImageCache();
```

Comment:
Just a note for possible follow-ups: if we start using this cache more often, we might want to consider renaming it since it's now used for more than just image caching.

Reply:
You're absolutely correct here; this was just me being lazy :-)

We could easily just have a base `class`, with Image/ColorSpace/etc. ones extending it as needed. If we want to cache by reference as well, which *may* help for the SMask code-path, then we probably need slightly different `set` methods anyway...

Reply:
This is now being fixed, among other things, in PR #12012 :-)